src/Radarr.Api.V3/ProviderResource.cs

```diff
@@ -38,8 +38,7 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Tags = definition.Tags,
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
 
-                InfoLink = string.Format("https://github.com/Radarr/Radarr/wiki/Supported-{0}#{1}",
-                    typeof(TProviderResource).Name.Replace("Resource", "s"),
+                InfoLink = string.Format("https://wiki.servarr.com/Radarr_Supported_{0}",
```

Comment:
Lets change this back to `{0}#{1}` for the new wiki... That way if we have a section dedicated to that specific Indexer/Client it will go straight there instead of the overall section.